warehouse/forklift/legacy.py

```diff
@@ -143,8 +143,8 @@ def namespace_stdlib_list(module_list):
     "11",
 }
 
-# manylinux pep600 is a little more complicated:
-_manylinux_platform_re = re.compile(r"manylinux_(\d+)_(\d+)_(?P<arch>.*)")
+# manylinux pep600 and musllinux pep656 are a little more complicated:
+_manylinux_re = re.compile(r"(?P<libc>(many|musl))linux_(\d+)_(\d+)_(?P<arch>.*)")
```

Comment:
Since "manylinux" is now used both as an overall umbrella, as well as for the platform-tag specifically, perhaps this could be:

```suggestion
_linux_platform_re = re.compile(r"(?P<libc>(many|musl))linux_(\d+)_(\d+)_(?P<arch>.*)")
```